pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1505,4 +1505,6 @@ self: super: {
   # Test suite requires running a database server. Testing is done upstream.
   hasql-pool = dontCheck super.hasql-pool;
 
+  headed-megaparsec = addBuildDepend super.headed-megaparsec super.selective_0_4_1;
```

Comment:
Please add a comment on this line of why this build depend is needed.

Reply:
Done